### DISCUSSION TITLE
Port conflict with docker-proxy

### DISCUSSION THREAD

**Discussion by rjwatsonrussell** [NginxProxyManager/nginx-proxy-manager#4209] [Q&A]
Hi all,

Apologies if this is a silly question, as I am new to Docker and nginx. I'm currently running Docker on a Raspberry Pi, and would like to install NginxProxyManager for the purpose of setting up reverse proxy in order to allow me to redirect domain names (plex.home, for example) to specific ports on my Raspberry Pi (192.168.xx.xx:32400). I'm running into an issue in that docker-proxy is currently using port 80 to allow web access to my containers which is causing a conflict when installing NginxProxyManager. I can, in theory set Nginx to not listen on port 80, however as I understand it, this would mean Nginx would not detect requests from a web browser and therefore not redirect to the desired IP and port. 

Is there any way around this or is it just not possible to run Docker and NginxProxyManager on the same server?

Thanks for your help.

**Comment by lastsamurai26** [comment]
If you don't want to use port 80 for the NPM, you can just do it. 
and only use 
ports:
443:443
81:81

If 80:80 is removed, the NPM can still start. 
It is then only important that all websites are accessed directly with https:.

For example, I have set nginx to 8080 and run everything via the NPM. 
If a local site is running on Nginx, it looks like this

Port 80 -> npm -> 8080 -> nginx 
Port 443 -> npm -> 8080 -> nginx
